src/Symfony/Component/Console/Application.php

```diff
@@ -335,6 +335,8 @@ public function register($name)
     /**
      * Adds an array of command objects.
      *
+     * If a Command is not enabled it will not be added.
```

Comment:
What do you think about using ``@see Application::add()`` to explain the behaviour of this method ?